extractors-opendronemap/terraref-opendronemap.py

```diff
@@ -94,6 +165,33 @@ def filename_sensor_maps(self):
     def sensor_name(self):
         return 'rgb_fullfield'
 
+    # pylint: disable=too-many-arguments
+    def update_dataset_extractor_metadata(self, connector, host, key, dsid, metadata,\
+                                          extractor_name):
+        """Adds or replaces existing dataset metadata for the specified extractor
+
+        Args:
+            connector(obj): the message queue connector instance
+            host(str): the URI of the host making the connection
+            key(str): used with the host API
+            dsid(str): the dataset to update
+            metadata(str): the metadata string to update the dataset with
+            extractor_name(str): the name of the extractor to associate the metadata with
+        """
+        meta = build_metadata(host, self.extractor_info, dsid, metadata, "dataset")
+
+        try:
+            md = ds.download_metadata(connector, host, key, dsid, extractor_name)
+            md_len = len(md)
+        except Exception as ex:     # pylint: disable=broad-except
```

Comment:
Ditto re narrowing.

Reply:
Same answer as above in update_dataset_extractor_metadata() in file terra_clipbyshape.py